### ISSUE TITLE
Request for plumbing comments through?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [CastXML/pygccxml#128]
Per the merge in https://github.com/CastXML/CastXML/pull/175, would it be possible to expose this here?

Similar to `clang.cindex` (relates #123), would it be possible to expose the API to get the source location (and possibly the source itself) for the comment?

See example usage here:
https://github.com/RobotLocomotion/drake/blob/6afa9365ad7d873c8c272fe8b5cc5a5c47ada6f8/tools/workspace/pybind11/mkdoc.py#L200-L201

**Comment by wabscale** [comment]
Do you know if this ever made it through? I can see places in this codebase that seem to reference support for comments, but the objects seem to always remain unpopulated.

See [src/pygccxml/declarations/comment.py](https://github.com/CastXML/pygccxml/blob/develop/src/pygccxml/declarations/comment.py)

**Comment by iMichka** [comment]
Yes it was implemented with https://github.com/CastXML/pygccxml/pull/130.

Do you have a reproducible example I could try to help debug your issue?

**Comment by iMichka** [comment]
(and I think we forgot to close this issue)
